src/ice_shelf/MOM_ice_shelf_initialize.F90

```diff
@@ -169,7 +169,7 @@ subroutine initialize_ice_thickness_from_file(h_shelf, area_shelf_h, hmask, G, U
 
         if (area_shelf_h(i,j) >= G%areaT(i,j)) then
           hmask(i,j) = 1.
-          area_shelf_h(i,j)=G%areaT(i,j)
+!          area_shelf_h(i,j)=G%areaT(i,j)
```

Comment:
Why would it make sense for the area covered by the shelf in a grid cell to exceed the area of the grid cell itself?

Reply:
`area_shelf_f` is read from a file, `G%areaT` is computed at the initialization of the horizontal grid. The two might not be identical.

Reply:
I think that it is fine that they are not identical, but the line that was commented out deals with this possibility by enforcing that the shelf_area_h can not exceed the area of the cell, which seems to me like a very natural restriction.  What would it mean if the ratio of the areas as (area_shelf_h/G%areaT) were greater than 1?

Reply:
The line is uncommented. The ratio of areas cannot be greater than 1.